<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Car Renting Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QTabWidget" name="Car_Tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="Rent_Tab">
       <attribute name="title">
        <string>Rent</string>
       </attribute>
       <layout class="QGridLayout" name="Rent_layout">
        <item row="0" column="1">
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="Rent">
           <attribute name="title">
            <string>Rent</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="1" column="0" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_32">
              <item>
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>FirstName</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_21">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="Rent_FirstName"/>
              </item>
             </layout>
            </item>
            <item row="1" column="3" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_33">
              <item>
               <widget class="QLabel" name="label_26">
                <property name="text">
                 <string>ID</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_22">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>55</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="Rent_ID"/>
              </item>
             </layout>
            </item>
            <item row="2" column="0" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_34">
              <item>
               <widget class="QLabel" name="label_27">
                <property name="text">
                 <string>Birthday</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_23">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>12</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="Rent_Birthday"/>
              </item>
             </layout>
            </item>
            <item row="0" column="3" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_31">
              <item>
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_20">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>31</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="Rent_Name"/>
              </item>
             </layout>
            </item>
            <item row="0" column="0" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Car License</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Rent_Car_License"/>
              </item>
             </layout>
            </item>
            <item row="2" column="5">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="Rent_Car_Button">
                <property name="text">
                 <string>Rent</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="3" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_35">
              <item>
               <widget class="QLabel" name="label_28">
                <property name="text">
                 <string>Telephone</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_24">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="Rent_Telephone"/>
              </item>
             </layout>
            </item>
           </layout>
           <zorder>horizontalSpacer_14</zorder>
           <zorder>layoutWidget</zorder>
           <zorder>horizontalLayoutWidget</zorder>
           <zorder>layoutWidget_2</zorder>
           <zorder>layoutWidget_3</zorder>
           <zorder>layoutWidget_4</zorder>
           <zorder>layoutWidget_5</zorder>
           <zorder></zorder>
          </widget>
          <widget class="QWidget" name="Return">
           <attribute name="title">
            <string>Return</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Car License</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Return_Car_License"/>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="Return_Car_Button">
                <property name="text">
                 <string>Return</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Car">
       <attribute name="title">
        <string>Car</string>
       </attribute>
       <layout class="QGridLayout" name="Car_layout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QTabWidget" name="car_tab">
            <property name="currentIndex">
             <number>4</number>
            </property>
            <widget class="QWidget" name="addCr">
             <attribute name="title">
              <string>Add</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="5" column="2">
               <widget class="QPushButton" name="Car_Add">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>HorsePower</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_9">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="Car_HP_Add"/>
                </item>
               </layout>
              </item>
              <item row="1" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Model</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="Car_Add_Model"/>
                </item>
               </layout>
              </item>
              <item row="2" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_18">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>License</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>34</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="Car_Add_License_"/>
                </item>
               </layout>
              </item>
              <item row="3" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_19">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Color</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>47</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="Car_Color_Add"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="Remove">
             <attribute name="title">
              <string>Remove</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <item>
                 <widget class="QPushButton" name="Car_Remove">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Car License</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Car_License_Remove"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="Available">
             <attribute name="title">
              <string>Available</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QTableView" name="available_cars"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="Cars">
             <attribute name="title">
              <string>Cars</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QTableView" name="all_cars"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="Search">
             <attribute name="title">
              <string>Search</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLineEdit" name="Search_field"/>
                </item>
                <item>
                 <widget class="QPushButton" name="Search_button">
                  <property name="text">
                   <string>Search</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTableView" name="Car_Search_Results"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Client">
       <attribute name="title">
        <string>Client</string>
       </attribute>
       <layout class="QGridLayout" name="Client_layout">
        <item row="0" column="0">
         <widget class="QTabWidget" name="Clients_tab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="Renters">
           <attribute name="title">
            <string>Renters</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTableView" name="renters_clients"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="Clients">
           <attribute name="title">
            <string>Clients</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QTableView" name="all_clients"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
